AmericaSCORESBayArea/Student-Registration-React-Web
# Feature/student web 62 input info modal

Things to test:
- Login as a coach and select the option to register a new student. Complete the form verifying that the email options are not mandatory. In the parent email field, verify that the information button appears correctly and that when you touch it, a modal opens with an icon, title, text, and button to close it.
- Try the same but with the option to edit student.
Devices to test on:
- Android
- IOS
- Mac OS
- Windows
